functions

```diff
@@ -79,11 +84,48 @@ service_create_container() {
   service_info "$SERVICE"
 }
 
+service_create_database() {
+  local SERVICE="$1"
+  local DATABASE="$2"
+  local SERVICE_ROOT="$PLUGIN_DATA_ROOT/$SERVICE"
+
+  [[ -n "$SERVICE" ]] || dokku_log_fail "Please specify a name for the service"
+  [[ -d "$SERVICE_ROOT" ]] || dokku_log_fail "$PLUGIN_SERVICE service $SERVICE does not exist"
+  [[ ! -e "$SERVICE_ROOT/databases/$DATABASE" ]] || dokku_log_fail "$PLUGIN_SERVICE service $SERVICE already has database $DATABASE"
+
+  local SERVICE_NAME="$(get_service_name "$SERVICE")"
+  local ROOTPASSWORD="$(cat "$SERVICE_ROOT/auth/root")"
+  local SERVICE_TTY_OPTS
+  has_tty && SERVICE_TTY_OPTS="-t"
+
+  local USERNAME=$(openssl rand -hex 8)
```

Comment:
note2: you could reuse service name for it + maybe random prefix/suffix if you want.